README.md

```diff
@@ -24,7 +24,7 @@ You can run every example with zero setup (no registration needed).
 
 | | Docs | Neptune | GitHub | Colab
 | ----------- | :---: | :---: | :------: | :---:
-| Quickstart | [![docs]](https://docs.neptune.ai/usage/quickstart/) | [![neptune]](https://app.neptune.ai/o/common/org/quickstarts/experiments) | [![github]](how-to-guides/hello-neptune/scripts/hello_neptune.py) | [![colab]](https://colab.research.google.com/github/neptune-ai/examples/blob/master/how-to-guides/hello-neptune/notebooks/hello_neptune.ipynb) |
+| Quickstart | [![docs]](https://docs.neptune.ai/usage/quickstart/) | [![neptune]](https://app.neptune.ai/o/common/org/quickstarts/runs/details?viewId=d48562e1-a494-4fd0-b3bb-078240516a4f&detailsTab=metadata&shortId=QUI-92233&type=run) | [![github]](how-to-guides/hello-neptune/scripts/hello_neptune.py) | [![colab]](https://colab.research.google.com/github/neptune-ai/examples/blob/master/how-to-guides/hello-neptune/notebooks/hello_neptune.ipynb) |
```

Comment:
Isn't this example run going to be rather short-lived? I feel like every time I've tried to access a specific `QUI-` run (that was also tagged as `showcase-run`) it has no longer been available

Reply:
The first one seems to be pointing to a seaborn example, and the second to a custom dashboard I never knew existed.
Lets discuss this on Slack